src/test/resources/packaging/scripts/25_tar_plugins.bats

```diff
@@ -342,3 +343,100 @@ setup() {
     run rm -rf "$TEMP_CONFIG_DIR"
     [ "$status" -eq 0 ]
 }
+
+@test "[TAR] install shield plugin to elasticsearch directory with a space" {
+    export ES_DIR="/tmp/elastic search"
+
+    # Install the archive
+    install_archive
+
+    # Checks that the archive is correctly installed
+    verify_archive_installation
+
+    # Move the Elasticsearch installation to a directory with a space in it
+    rm -rf "$ES_DIR"
+    mv /tmp/elasticsearch "$ES_DIR"
+
+    # Checks that plugin archive is available
+    [ -e "$SHIELD_ZIP" ]
+
+    # Install Shield
+    run "$ES_DIR/bin/plugin" -i elasticsearch/shield/latest -u "file://$SHIELD_ZIP"
+    [ "$status" -eq 0 ]
+
+    # Checks that Shield is correctly installed
+    assert_file_exist "$ES_DIR/bin/shield"
+    assert_file_exist "$ES_DIR/bin/shield/esusers"
+    assert_file_exist "$ES_DIR/bin/shield/syskeygen"
+    assert_file_exist "$ES_DIR/config/shield"
+    assert_file_exist "$ES_DIR/config/shield/role_mapping.yml"
+    assert_file_exist "$ES_DIR/config/shield/roles.yml"
+    assert_file_exist "$ES_DIR/config/shield/users"
+    assert_file_exist "$ES_DIR/config/shield/users_roles"
+    assert_file_exist "$ES_DIR/plugins/shield"
+
+    # Remove the plugin
+    run "$ES_DIR/bin/plugin" -r elasticsearch/shield/latest
+    [ "$status" -eq 0 ]
+
+    # Checks that the plugin is correctly removed
+    assert_file_not_exist "$ES_DIR/bin/shield"
+    assert_file_exist "$ES_DIR/config/shield"
```

Comment:
> $ES_DIR/config is intentionally not removed? might be worth a comment in the test

Configuration is not removed when removing a plugin. I think elasticsearch configuration is removed when purging the package.